setup.py

```diff
@@ -4,7 +4,7 @@
 long_description = open('README.md', 'r').read()
 
 install_requires = [
-    'wagtail>=1.13,<2.9',
+    'wagtail>=2.2,<2.9',
```

Comment:
Because we're testing 2.3 and 2.8, could we make this 2.2?
```suggestion
    'wagtail>=2.3,<2.9',
```

Reply:
Good catch